lit-config.js

```diff
@@ -46,23 +46,30 @@ module.exports = {
 	"overrides": [{
 		"files": "./**/lang/*.js",	
 		"rules": {
-			"quotes": 0
+			"quotes": "off",
+			"key-spacing": "error",
```

Comment:
Minor: Do we want to use integers to match the rest of the file?

Reply:
I can yeah, although I find the integers difficult to remember which is which.

Reply:
Agreed, this is easier to read lol, I just like consistency a bit more. We could switch them all - we need a linter for our lint configs 😝